### PR TITLE
Fixing spelling mistakes

### PR DESCRIPTION
**Pull request by mvvsubbu**

### CODE REVIEW

File: Tasks/AzureFileCopy/IAzureUtility.ps1 [hunk 1]
```diff
@@ -93,7 +93,7 @@ function Get-AzureMachineStatus
           [string]$name)
 }
 
-# Returns details of the custon script extension $name of VM $vmName present in ResourceGroup $resourceGroupName
+# Returns details of the customscript extension $name of VM $vmName present in ResourceGroup $resourceGroupName
```
Comment (KrishnaAdityaB) [inline Tasks/AzureFileCopy/IAzureUtility.ps1]: customscript is not a single word.

File: Tasks/AzureFileCopy/task.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 50
```
Comment (KrishnaAdityaB) [inline Tasks/AzureFileCopy/task.json]: No customer facing fixes in this task. should we bump up the task version?

File: Tasks/AzureFileCopy/Utility.ps1 [hunk 3]
```diff
@@ -14,7 +14,7 @@ $telemetryCodes =
   "PREREQ_InvalidFilePath" = "PREREQ_InvalidFilePath";
   "PREREQ_StorageAccountNotFound" = "PREREQ_StorageAccountNotFound";
   "PREREQ_NoVMResources" = "PREREQ_NoVMResources";
-  "PREREQ_UnsupportedAzurePSVerion" = "PREREQ_UnsupportedAzurePSVerion";
```
Comment (KrishnaAdityaB) [inline Tasks/AzureFileCopy/Utility.ps1]: Is it conscious? our telemetry queries may break.

Comment (mvvsubbu) [inline Tasks/AzureFileCopy/Utility.ps1]: Till now our telemetry results were not coming at all so i thought changing now is not a problem

File: Tasks/AzureFileCopy/task.json (hunk 2 above)
Comment (mvvsubbu) [inline Tasks/AzureFileCopy/task.json]: yes.. but compact tool fails

File: Tasks/AzureFileCopy/IAzureUtility.ps1 (hunk 1 above)
Comment (mvvsubbu) [inline Tasks/AzureFileCopy/IAzureUtility.ps1]: ok i will fix it

**Pull request merged**